docs/src/test-api/class-testconfig.md

```diff
@@ -232,17 +232,34 @@ export default defineConfig({
 
 ## property: TestConfig.metadata
 * since: v1.10
-- type: ?<[Metadata]>
-
-Metadata that will be put directly to the test report serialized as JSON.
+- type: ?<[Object]>
+  - `revision.id` ?<[string]> The unique identifier (e.g., hash or GUID) for the revision or commit that triggered the test run.
```

Comment:
These are not actually populated

Reply:
@pavelfeldman could you elaborate more, what do you mean under "not actually populated"?
I've tried with the latest Playwright - all these fields are shown in the report:
![image](https://github.com/user-attachments/assets/487c2099-ff8e-4b61-b577-6b540e472afa)